app/src/main/res/resource/values-de/strings.xml

```diff
@@ -154,30 +156,30 @@
     <string name="feedback_exceed_rate_limit">Ups! Es gab zu viele Besucher in dieser Stunde, bitte versuchen Sie in der nächsten Stunde erneut.</string>
 
     <string name="feedback_confirm_change_theme">Themaänderung bestätigen</string>
-    <string name="feedback_change_theme_warning">Jetztiges Ändern des Themas wird aktuelle downloads abbrechen.</string>
+    <string name="feedback_change_theme_warning">Änderung des Themas wird alle aktuelle Downloads abbrechen.</string>
     <string name="feedback_confirm_restart">Neustart bestätigen</string>
-    <string name="feedback_restart_warning">Jetztiges Neustarten der App wird alle Downloads abbrechen.</string>
+    <string name="feedback_restart_warning">Neustarten der App wird alle aktuellen Downloads abbrechen.</string>
 
     <string name="feedback_click_again_to_exit">Zum Verlassen nochmal drücken.</string>
 
     <string name="feedback_share_photo_title">Ein Foto von Unsplash.com</string>
-    <string name="feedback_share_photo_extra">Genommen von # am #</string>
+    <string name="feedback_share_photo_extra">Genommen von # am #\n</string>
     <string name="feedback_share_collection_title">Eine Kollektion von Unsplash.com</string>
-    <string name="feedback_share_collection_extra">Erstellt von # am #</string>
+    <string name="feedback_share_collection_extra">Erstellt von # am #\n</string>
     <string name="feedback_share_user_title">Ein Benutzerprofil von Unsplash.com</string>
-    <string name="feedback_share_user_extra">#</string>
+    <string name="feedback_share_user_extra">#\n</string>
 
-    <string name="feedback_like_failed">Liken des Fotos fehlgeschlagen.</string>
-    <string name="feedback_unlike_failed">Unliken des Fotos fehlgeschlagen.</string>
+    <string name="feedback_like_failed">Favorisieren des Fotos fehlgeschlagen.</string>
+    <string name="feedback_unlike_failed">Als Favorit entfernen fehlgeschlagen.</string>
 
     <string name="feedback_follow_failed">Folgen fehlgeschlagen.</string>
-    <string name="feedback_cancel_follow_failed">Unfolgen fehlgeschlagen.</string>
+    <string name="feedback_cancel_follow_failed">Nicht mehr folgen ist fehlgeschlagen.</string>
 
     <string name="feedback_story_is_null">Geschichte existiert nicht oder wurde noch nicht geladen.</string>
 
     <!-- introduce. -->
     <string name="introduce_title_back_top">Nach oben scrollen</string>
-    <string name="introduce_description_back_top">Tippen Sie auf die TopBar uder Zurück, die Fotolistenansicht wird zurück nach oben scrollen.</string>
+    <string name="introduce_description_back_top">Tippen Sie auf die TopBar oder Zurück, um in der Listenansicht nach oben zu scrollen.</string>
 
```

Comment:
Missing: `<string name="muzei_description">Featured Photos from Unsplash</string>`

"Empfohlene Fotos von Unsplash" (or when it is meant like that: "Von Unsplash empfohlene/vorgeschlagene Fotos"